packages/fxa-content-server/app/scripts/lib/experiments/grouping-rules/cwts-on-signup-password.js

```diff
@@ -10,7 +10,7 @@ const EXPERIMENT_NAME = 'signupPasswordCWTS';
 
 const GROUPS = ['control', 'treatment'];
 
-const ROLLOUT_RATE = 0.2;
+const ROLLOUT_RATE = 1;
```

Comment:
This works because of [this branch here](https://github.com/mozilla/fxa/pull/3466/files#diff-d6caf564bc7e3f6c366d27554185d1f4R42).